client/src/services/User.js

```diff
@@ -56,6 +56,32 @@ export function handleLogin(data, history) {
     });
 }
 
+export function getUserData() {
+  fetch(`${BASE_URL}/auth/user`, {
+    headers: {
+      Authorization: `Token ${localStorage.getItem("token")}`
+    }
+  })
+    .then(res => res.json())
+    .then(json => {
+      // console.log(json);
```

Comment:
Delete any commented out stuff throughout the PR

Reply:
done